approval_polls/static/approval_polls/create.js

```diff
@@ -18,4 +18,35 @@ $(function () {
 
   $('[data-toggle=tooltip]').tooltip();
   $('button#add-choice').click($.proxy(this.addChoiceField, this));
+
+  function roundMinutes(date) {
```

Comment:
Quite minor, but it is better to move all variable declarations to the top because the JavaScript interpreter will move them up anyway. See [this blog post on hoisting](http://www.adequatelygood.com/JavaScript-Scoping-and-Hoisting.html). Also to keep it consistent with the other function definitions in the file. `roundMinutes = function(date) { ... } `